src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/ptv/WebpageServiceChannelInTestsIT.java

```diff
@@ -0,0 +1,133 @@
+    getPtvOrganizationMocker().mock(TestPtvConsts.ORGANIZATIONS);
+    getPtvServiceMocker().mock(TestPtvConsts.SERVICES);    
+    getPtvServiceChannelMocker().mock(TestPtvConsts.SERVICE_CHANNELS);
+
+    startMocks();
+    
+    waitApiListCount("/organizations", 3);
+    waitApiListCount("/webPageServiceChannels", TestPtvConsts.WEB_PAGE_SERVICE_CHANNELS.length);
+    waitApiListCount("/services", TestPtvConsts.SERVICES.length);
+  }
+  
+  @Test
+  public void updateWebPageServiceChannelUnauthorized() throws JsonParseException, JsonMappingException, IOException, InterruptedException {
+    WebPageServiceChannel kuntaApiResource = getWebPageChannel(0, TestPtvConsts.WEB_PAGE_SERVICE_CHANNELS.length);
+    
+    givenReadonly()
+      .body(kuntaApiResource)
+      .contentType(ContentType.JSON)
+      .put("/webPageServiceChannels/{kuntaApiChannelId}", kuntaApiResource.getId())
+      .then()
+      .assertThat()
+      .statusCode(401);
+  }
+  
+  @Test
+  public void updateWebPageServiceChannelForbidden() throws JsonParseException, JsonMappingException, IOException, InterruptedException {
+    WebPageServiceChannel kuntaApiResource = getWebPageChannel(0, TestPtvConsts.WEB_PAGE_SERVICE_CHANNELS.length);
+    
+    givenReadWrite()
+      .body(kuntaApiResource)
+      .contentType(ContentType.JSON)
+      .put("/webPageServiceChannels/{kuntaApiChannelId}", kuntaApiResource.getId())
+      .then()
+      .assertThat()
+      .statusCode(403);
+  }
+  
+  @Test
+  public void updateWebPageServiceChannelUnchanged() throws JsonParseException, JsonMappingException, IOException, InterruptedException {
+    String ptvId = TestPtvConsts.WEB_PAGE_SERVICE_CHANNELS[0];
+    String organizationId = getOrganizationId(0);
+    String kuntaApiChannelId = getWebPageChannelId(0, TestPtvConsts.WEB_PAGE_SERVICE_CHANNELS.length);
+    
+    grantOrganizationPermission(AccessType.READ_WRITE, organizationId, ClientOrganizationPermission.UPDATE_SERVICE_CHANNELS);
+
+    WebPageServiceChannel kuntaApiResource = getWebPageChannel(0, TestPtvConsts.WEB_PAGE_SERVICE_CHANNELS.length);
+    V6VmOpenApiWebPageChannelInBase ptvInResource = getPtvServiceChannelMocker().readEntity(AbstractPtvMocker.PTV_IN_API, ptvId, V6VmOpenApiWebPageChannelInBase.class);
+    V7VmOpenApiWebPageChannel ptvOutResource =  getPtvServiceChannelMocker().readEntity(AbstractPtvMocker.PTV_OUT_API, ptvId, V7VmOpenApiWebPageChannel.class);
+    
+    getPtvServiceChannelMocker().mockWebpagePut(ptvId, ptvOutResource);
+    
+    givenReadWrite()
+      .body(kuntaApiResource)
+      .contentType(ContentType.JSON)
+      .put("/webPageServiceChannels/{kuntaApiChannelId}",kuntaApiChannelId)
+      .then()
+      .assertThat()
+      .statusCode(200);
+
+    getPtvServiceChannelMocker().verifyWebPage(ptvId, ptvInResource);
+  }
+  
+  @Test
+  public void updateWebPageServiceChannelChanges() throws JsonParseException, JsonMappingException, IOException, InterruptedException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)